### ISSUE TITLE
Feature: Add support for language 

### ISSUE THREAD

**Issue by wetgi** [october-widgets/address#4]
Example:
http://maps.googleapis.com/maps/api/js?libraries=places&sensor=false&&language=[de, en, fr, ...]

It would be nice to define the language in the yaml file.

**Comment by scottbedard** [comment]
I agree this would be useful. I'm busy with other projects right now so I don't know when I'll have time to add this, but I would happily accept a pull request. Just make sure to send it to [this repository](https://github.com/october-widgets/library).
